presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -105,6 +108,11 @@
 
     void revokeRoles(Set<String> roles, Set<HivePrincipal> grantees, boolean adminOption, HivePrincipal grantor);
 
+    default Set<RoleGrant> listPrincipals(String role)
```

Comment:
remove `default`? Other methods do not have `default` here?

Reply:
New update soon.